src/wp-includes/block-patterns.php

```diff
@@ -328,6 +328,17 @@ function _register_remote_theme_patterns() {
  * @access private
  */
 function _register_theme_block_patterns() {
+
+	/**
```

Comment:
I meant for https://github.com/WordPress/wordpress-develop/pull/5568#pullrequestreview-1698313737 to be included here...sorry